extensions/shared/videopress-plan-check.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import './videopress-plan-check.scss';
+import UpgradeNudge from './components/upgrade-nudge';
+import getJetpackExtensionAvailability from './get-jetpack-extension-availability';
+//import isSimpleSite from './site-type-utils';
```

Comment:
I will re-enable this check once this is ready to merge. If it's preferred to do this now, I can also do that too.